app/src/main/java/org/mozilla/reference/browser/ext/View.kt

```diff
@@ -0,0 +1,43 @@
+/*
+ * This Source Code Form is subject to the terms of the Mozilla Public
+ * License, v. 2.0. If a copy of the MPL was not distributed with this
+ * file, You can obtain one at http://mozilla.org/MPL/2.0/.
+ */
+
+package org.mozilla.reference.browser.ext
+
+import android.view.View
+import android.view.ViewGroup
+import android.widget.FrameLayout
+import androidx.coordinatorlayout.widget.CoordinatorLayout
+
+// Implementation is from the default `Snackbar`
+@Suppress("ComplexMethod")
+internal fun View?.findSuitableParent(): ViewGroup? {
```

Comment:
If we need this for the snackbar, why is this not in our DaisySnackbar?

Reply:
Make sense. Moved to the snackbar class